src/minter-suite-mapping.ts

```diff
@@ -692,54 +680,30 @@ export function handleUnregisteredNFTAddressV2(
 // Because AssemblyScript does not support union types, we need to manually type check inside each method, to ensure correct usage.
 // For any questions reach out to @jon or @ryley-o.eth. or see the following document https://docs.google.com/document/d/1XSxl04eJyTxc_rbj6cmq-j00zaYDzApBBLT67JXtaOw/edit?disco=AAAAZa8xp-Q
 
-export function handleSetValueGeneric<T, C>(
-  event: T,
-  config: C,
-  project: Project | null
+export function handleSetMinterDetailsGeneric<T, C>(
+  key: string,
+  value: T,
+  config: C
 ): void {
-  if (
-    !(
-      event instanceof ConfigValueSetBool ||
-      event instanceof ConfigValueSetBigInt ||
-      event instanceof ConfigValueSetAddress ||
-      event instanceof ConfigValueSetBytes
-    )
-  ) {
-    return;
-  }
-
-  if (
-    !(config instanceof ProjectMinterConfiguration || config instanceof Minter)
-  ) {
-    return;
-  }
-
-  if (project) {
-    project.updatedAt = event.block.timestamp;
-  }
-
   let minterDetails = getMinterDetails(config);
-  let jsonKey = event.params._key.toString();
+  let jsonKey = key;
   let jsonValue: JSONValue;
 
-  if (event instanceof ConfigValueSetBool) {
-    jsonValue = json.fromString(booleanToString(event.params._value));
-  } else if (event instanceof ConfigValueSetBigInt) {
-    jsonValue = json.fromString(event.params._value.toString());
-  } else if (event instanceof ConfigValueSetAddress) {
-    jsonValue = stringToJSONValue(event.params._value.toHexString());
-  } else if (event instanceof ConfigValueSetBytes) {
-    jsonValue = bytesToJSONValue(event.params._value);
+  if (isBoolean(value)) {
+    jsonValue = json.fromString(booleanToString(value));
+  } else if (value instanceof BigInt) {
+    jsonValue = json.fromString(value.toString());
+  } else if (value instanceof Address) {
+    jsonValue = stringToJSONValue(value.toHexString());
+  } else if (value instanceof Bytes) {
+    jsonValue = bytesToJSONValue(value);
+  } else {
+    return;
```

Comment:
nit: we should log a warning here, since we don't expect to ever reach this point.